.npmignore

```diff
@@ -10,11 +10,14 @@
 /.editorconfig
 /.ember-cli
 /.env*
+/.eslintcache
 /.eslintignore
 /.eslintrc.js
 /.git/
 /.github/
 /.gitignore
+/.prettierignore
```

Comment:
These two are not yet part of the latest blueprint but a PR for this was merged recently:
https://github.com/ember-cli/ember-cli/pull/9437